src/lib/prov/tpm/tpm.cpp

```diff
@@ -359,7 +359,7 @@ std::vector<byte> TPM_PrivateKey::x509_subject_public_key() const
       .get_contents_unlocked();
    }
 
-secure_vector<byte> TPM_PrivateKey::pkcs8_private_key() const
+secure_vector<byte> TPM_PrivateKey::private_key_bits() const
    {
    throw TPM_Error("PKCS #8 export not supported for TPM keys");
```

Comment:
Message has to be adjusted. `export not supported for TPM private keys` or something similar.